tests/check/check_h26xsigned_auth.c

```diff
@@ -1814,9 +1813,8 @@ generate_and_set_private_key_on_camera_side(struct sv_setting setting,
   SignedVideoReturnCode sv_rc;
   char *private_key = NULL;
   size_t private_key_size = 0;
-  //  signed_video_nalu_to_prepend_t nalu_to_prepend = {0};
   nalu_list_item_t *i_nalu = nalu_list_item_create_and_set_id('I', 0, setting.codec);
-  size_t data_size;
+  size_t sei_size;
```

Comment:
Declare this where you use it, or move the declaration of `sei` (former `nalu_data`) here.